### ISSUE TITLE
About page isn't translated yet although some languages are supposed to be 100% translated

### ISSUE THREAD

**Issue by JustSpook** [searxng/searxng#1687]
**Is your feature request related to a problem? Please describe.**

I am a user of the engine and I use the Spanish language, everything is well translated but when I go to the about and search syntax page they are not translated. **Describe the solution you'd like**

It would be helpful if it could detect the language you are in and translate both pages as well.

**Comment by return42** [comment]
Hi @JustSpook: the translation of the [info-pages](https://github.com/searxng/searxng/tree/master/searx/infopage) is independent from the translation messages we host on https://weblate.bubu1.eu/projects/searxng/searxng/es/

We are looking for translators on weblate and for the info-pages, do you like to translate the info-pages to ES? .. For this just copy the [en](https://github.com/searxng/searxng/tree/master/searx/infopage/en) folder to `es` and translate the pages in the new created `es` folder.

If you think translation is done, send us your PR .. thanks!

**Comment by return42** [comment]
Still waiting for translations .. there is nothing we can do without contribution from translators .. I think the question is answered / going to close this issue.

**Comment by JustSpook** [comment]
Yes, the question is answered, I am very sorry that I could not contribute in this case. When I reported the incident I had, I didn't have much time to help and when I had time I forgot about this issue. I hope you can find help soon. Greetings and continue with the project, it's really good!
